ncm-metaconfig/src/main/metaconfig/icinga-web/pan/schema.pan

```diff
@@ -0,0 +1,41 @@
+declaration template metaconfig/icinga-web/schema;
+
+include 'pan/types';
+
+type iw_caching_opts = {
+    'enabled' : boolean
+    'driver' : string with match (SELF, "^(apc|memcache)$")
+    'use_query_cache' : boolean
+    'use_result_cache' ? boolean
+    'result_cache_lifespan' ? long(0..) = 60
+};
+
+type icinga_database_dsn = {
+    'protocol' : string with match (SELF, "^(pgsql|mysql|oracle)$")
```

Comment:
The documentation also mentions `icingaOracle` but isn't really clear about what it is for...

> Special implementation that must be used when using an Icinga database (some table names are different, and name length limits must be automatically respected to ensure no developers are killing themselves during development).